oe_editorial.libraries.yml

```diff
@@ -0,0 +1,7 @@
+help:
```

Comment:
I think that that this js file in fact modification of drupal.filter library's js we have to use hook alter for libraries.

Reply:
We discussed with @sergepavle and we are adding extra js functionality so we can't extend the original.